cql/the-basics-of-cql/cinchy-supported-functions/connections-functions.md

```diff
@@ -4,37 +4,79 @@
 
 The set of functions listed in this page are for use in the [Cinchy Connections Experience](https://cli.docs.cinchy.com/) when configuring data syncs.
 
+* [@COLUMN]
 * [​JSON\_ESCAPE​](connections-functions.md#json\_escape)
 * [​URL\_ESCAPE​](connections-functions.md#url\_escape)
 * [STRING\_ESCAPE()](connections-functions.md#string\_escape)
+* [@ID](connections-functions#id)
+
+## @COLUMN
+
+The `@COLUMN` function is used for real-time syncs where your source is an event (such as Cinchy Event Broker) and your target is REST/SOAP. 
+
+Use `@COLUMN` to build the request body of SOAP/REST API specifications.
+
+### Syntax
+
+```
+{
+    "employeeNane": "@COLUMN('Name')"
+}
+```
+
+### Example
+
+The following example uses the Cinchy Event Broker as a Source with a REST API destination. 
+
+This scenario maps the data from the `Name` property to the REST API destination property `employeeName`.
+
+1. In the **Source** tab, set the source to Cinchy Event Broker. 
```

Comment:
Maybe make this a Cinchy Table to keep things simple if they want to try it out themselves? A CDC is harder to set up

Reply:
Changed to fit Cinchy Table.